docs/software/exportformat.md

```diff
@@ -60,6 +60,34 @@ are still included in reporting to provide access to historic data. Deprecated c
 
+
+## FutureVoter Tool Export Format
+
+| Column Title                             | Description                                                                                                                                                                                                                                    |
+|------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ID                                       | The ID we store internally for the registration                                                                                                                                                                                                |
+| Subscriber                               | The name of the subscriber                                                                                                                                                                                                                     |
+| Time Started (UTC)                       | The time that the first page of the registration was submitted                                                                                                                                                                                 |
+| First Name                               | The user's first name                                                                                                                                                                                                                          |
+| Date of Birth                            | The user's date of birth                                                                                                                                                                                                                       |
+| Zipcode                                  | The user's zipcode                                                                                                                                                                                                                             |
+| State                                    | The user's state                                                                                                                                                                                                                               |
+| Subscriber SMS Opt In                    | True/False if the user has selected to opt-in to the subscriber's SMS list                                                                                                                                                                     |
```

Comment:
If we're not providing phone number, should we provide this in the docs?

Reply:
Good point. Removed.